@xen-orchestra/lite/src/components/host/network/HostPifsTable.vue

```diff
@@ -90,7 +90,7 @@
                 size="small"
               />
               <div v-else-if="column.id === 'status'" v-tooltip>
```

Comment:
```suggestion
              <div v-else-if="column.id === 'status'">
```
v-tooltip is useless here